geode-core/src/main/java/org/apache/geode/internal/cache/TXStateProxy.java

```diff
@@ -64,14 +64,14 @@
   public TXSynchronizationRunnable getSynchronizationRunnable();
 
   /**
-   * Called by {@link TXManagerImpl#internalSuspend()} to perform additional tasks required to
-   * suspend a transaction
+   * Called by {@link TXManagerImpl#internalSuspend(boolean)} to perform additional tasks required
```

Comment:
Is this javadoc still accurate? I'm not sure it makes sense to javadoc who calls a method. The IDE will always tell you that. Consider changing this javadoc to just say "Performs any additional tasks required by the proxy to suspend a transaction"